website/src/_posts/whats-up-with-template-covariant.md

```diff
@@ -78,3 +78,5 @@ To summarize:
 
 * `@template` declares an invariant type variable: The object `Collection<Animal>` accepts only another `Collection<Animal>`. But the collection can be mutable and the type variable can be present in a parameter position. [Playground example »](https://phpstan.org/r/81513715-c26f-4a25-9709-a956d6d3e02b)
 * `@template-covariant` declares a covariant type variable: The object `Collection<Animal>` also accepts `Collection<Cat>`, but the type variable cannot be present in a parameter position. [Playground example »](https://phpstan.org/r/d2f62e2c-52fc-4956-87ea-fc4c8d481384)
+
+There is also an elegant way to have an invariant `Collection` but still be able to accept `Collection<Dog>` as well as `Collection<Animal>` where necessary. Learn more in the [guide to call-site variance](/blog/guide-to-call-site-generic-variance).
```

Comment:
I feel like this would deserve a heading and a couple of paragraph here at least, otherwise it's easy to miss :)